### ISSUE TITLE
Ummmm 520

### ISSUE THREAD

**Issue by herohamp** [technoboy10/crossorigin.me#79]
http://image.prntscr.com/image/863ef1792b824cb2aaf5b7c76de4a27f.png

All I did was

$.get("https://crossorigin.me/http://zorgtech.org", function( data ) {console.log(data)});

**Comment by technoboy10** [comment]
520 means the servers are down. Sorry about that, limited by budget at this point. :/

**Comment by herohamp** [comment]
Why do you use digitalocean

**Comment by technoboy10** [comment]
Best option whose TOS didn't prevent me from doing this. Going to change hosting soon though.

**Comment by herohamp** [comment]
ok what about this post you made?
![image](https://cloud.githubusercontent.com/assets/8162992/23088717/3ee11018-f54d-11e6-8fe7-08909ee5ec93.png)

**Comment by technoboy10** [comment]
tl;dr BJC paid for all of the Digital Ocean costs. I didn't plan on the site continuing to grow, so the support wasn't quite enough.

**Comment by herohamp** [comment]
oh ok

**Comment by herohamp** [comment]
I don't want to be to annoying ;P but what hosting company are you thinking of going to?

**Comment by technoboy10** [comment]
Can't say just yet. :)
